uint8 id

int16 pull_air_velocity
int16 pull_air_position
int16 pull_air_max_torque
int16 pull_air_hold_torque
---
uint8 state
